### DISCUSSION TITLE
Error: Mach correction requested on Ma = 1.248658200084483 >= 1.0

### DISCUSSION THREAD

**Discussion by HSP100** [byuflowlab/FLOWUnsteady#77] [Q&A]
I am getting this error while running the simulation.. how should I solve this error?
![WhatsApp Image 2023-06-27 at 2 40 49 PM](https://github.com/byuflowlab/FLOWUnsteady/assets/105547702/411baf42-f1b4-4e2b-bc54-aa21d22078ef)

**Comment by vincinu** [comment]
I have the same error, did you manage to solve?

**Reply by HSP100** [reply]
No, I am still trying to solve it.

**Comment by EdoAlvarezR** [comment]
This error is exactly what it says: the simulation detected that the velocity along the blade is supersonic, which could be cause by the user using an RPM that leads to a supersonic rotational velocity, or that the simulation blew up.

If you pull up the simulation (see [docs](https://flow.byu.edu/FLOWUnsteady/visualization/)), you can visually check if the simulation blew up. Also, check on the [enstrophy monitor](https://flow.byu.edu/FLOWUnsteady/api/flowunsteady-monitor/#FLOWUnsteady.generate_monitor_enstrophy) for numerical blow up.

**Reply by HSP100** [reply]
How to solve the error if it is because of simulation "blew up" and not because of "mach number > 1"?

**Comment by weijunsjtu** [comment]
I'm having the same problem, have you solved this one yet?

**Reply by EdoAlvarezR** [reply]
@weijunsjtu, this is the generic error that you run into when the simulation has blown up (the flow becomes supersonic), so I doubt your problem is the same one that @HSP100 ran into. I'd recommend visualizing the simulation to find out what is going on (see my previous comment).

**Reply by jayjiejie** [reply]
Hi Edo

Just wonder if there is any generic approach for the Mach correction issue?

Thank you.
Jay
